docs/software.md

```diff
@@ -83,6 +83,8 @@
 
 - [**Python - high-level interpreted programming language**](../software/programming/#python)
 - [**Docker - Build, ship, and run distributed applications**](../software/programming/#docker)
+- [**Docker-Compose - run multi-container applications on Docker**](../software/programming/#docker-compose)
```

Comment:
Start with capital letters after the '-': "run multi-container..." -> "Run multi-containter..."

Please change this also at the other entries: 
- Python (above)
- OpenTyrian (below)

Reply:
done